server/cli/start.go

```diff
@@ -169,6 +160,7 @@ func runStart(cmd *commander.Command, args []string) {
 	select {
 	case <-stopper.ShouldStop():
 		stopper.SetStopped()
+		close(gracefulShutdownCh)
```

Comment:
@spencerkimball in my commits to #636 I had the first option implemented and working -- basically just holding the lock in `Stop()` and doing nothing if the thing's state indicated it was already done.